app/components/dns-records-table.tsx

```diff
@@ -108,31 +109,33 @@ export default function DnsRecordsTable(props: DnsRecordsTableProps) {
                     <Td>
                       <Flex justifyContent="space-between" alignItems="center">
                         <DnsRecordName dnsRecord={dnsRecord} baseDomain={baseDomain} />
-                        <Link
-                          href={`https://dnschecker.org/#${dnsRecord.type}/${dnsRecord.subdomain}.${baseDomain}`}
-                          isExternal
-                          target="_blank"
-                        >
-                          <Tooltip label="Check DNS Record">
+                        <ButtonGroup>
+                          <Link
+                            href={`https://dnschecker.org/#${dnsRecord.type}/${dnsRecord.subdomain}.${baseDomain}`}
+                            isExternal
+                            target="_blank"
+                          >
+                            <Tooltip label="Check DNS Record">
+                              <IconButton
+                                icon={<InfoOutlineIcon color="black" boxSize="5" />}
+                                aria-label="Check DNS record"
+                                variant="ghost"
+                                ml="2"
+                              />
+                            </Tooltip>
+                          </Link>
+                          <Tooltip label="Copy subdomain to clipboard">
```

Comment:
Let's harmonize our naming/cases.  Above you have a tooltip label of `Check DNS Record` so let's say `Copy DNS Record to clipboard` or something.